data/language/en-GB.txt

```diff
@@ -4125,7 +4125,7 @@ STR_5814    :{WINDOW_COLOUR_1}{OPENQUOTES}{STRING}{ENDQUOTES}
 #tooltips
 STR_5815    :{SMALLFONT}{BLACK}Display FPS counter in-game
 STR_5816    :{SMALLFONT}{BLACK}Sets ratio to scale the game by.{NEWLINE}Most useful when playing in{NEWLINE}high resolutions
-STR_5817    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Sets UI scaling type
+STR_5817    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Sets UI scaling type{NEWLINE}{NEWLINE}Nearest Neighbour is sharp but causes distortion at non-integer values.{NEWLINE}Linear and Anisotropic are smooth but blurry.{NEWLINE}Smooth Nearest Neighbour is sharp with smoothing at non-integer values.
```

Comment:
It should mention SNN incurs additional cost. It's possibly minor but it's still there.